tests/test_vcs_lambert_azimuthal.py

```diff
@@ -0,0 +1,17 @@
+import basevcstest
+
+class TestVCSLambertAzimuthal(basevcstest.VCSBaseTest):
+    def allTests(self, centerlongitude):
```

Comment:
@danlipsa I think nose/unittest runs every function with `test` in its name. I would recommend renaming this to `main` or `core` or something like that.